### DISCUSSION TITLE
Docs input number helper

### DISCUSSION THREAD

**Discussion by Xander-V** [frenck/spook#478] [Documentation]
It seems the documentation is still describing a few older helper services regarding input_number what have been already fully integrated in the standard home assistant:

- input_number.decrement (spook: Decrease value 👻)
- input_number.increment (spook: Increase value 👻)

Solution:
Remove descriptions of no longer spook special entries, so only set minimum and set maximum should remain [here](https://spook.frenck.dev/input-number)

![image](https://github.com/frenck/spook/assets/97856913/9b3113c4-7d66-440f-8480-d490466fce8c)

**Comment by frenck** [comment]
> It seems the documentation is still describing a few older helper services regarding input_number what have been already fully integrated in the standard home assistant:


That statement isn't correct. They are different service that do different things (not incrementing or decrementing at all).

../Frenck

**Reply by Xander-V** [reply]
In that case I simply don't understand the docs :)
Can you elaborate? how come I see only two spook icons when looking for services regarding input_number?

**Reply by frenck** [reply]
![CleanShot 2023-12-11 at 12 07 41@2x](https://github.com/frenck/spook/assets/195327/8b60af97-02e8-4bdf-a5fe-983476c8454a)

**Reply by Xander-V** [reply]
Ahh, i see 👍 Thanks. (Begrijpend lezen is ook een vak...)

So you've changed the standard service to also accept an optional amount instead of always using the standard or set step size... Wouldn't it be more obvious then to just add a new service name with the spook icon attached to it, something like .increment_by?

For now then the only thing left then might be to update the first picture since the spook icons seem to be no longer there?

**Reply by frenck** [reply]
> Wouldn't it be more obvious then to just add a new service name with the spook icon attached to it, something like .increment_by?

Choices 🤷
